### ISSUE TITLE
Add an option to pick the latest .NET nightly SDK

### ISSUE THREAD

**Issue by Sergio0694** [ashmind/SharpLab#568]
Follow up from this comment from @tannergooding in the C# server:

> I imagine someone could contribute a PR that allows you to pick the latest .NET 5 Nightly SDK, acquired via dotnet-install scripts

I'm personally not able to contribute this feature myself, but opening this issue to track the proposal.
Sharplab currently lets you switch between some fixed SDK branches and some Roslyn branches, but it's missing a branch to just use the latest available .NET SDK. That would be incredibly useful both to be able to try out new APIs before an official release is published, as well as to check out the various updates to the JIT. For instance, opening issues in `dotnet/runtime` right now usually means you have to clone/build the whole runtime in order to be able to use [Disasmo](https://github.com/EgorBo/Disasmo) to verify that a given JIT result seen in sharplab is still present and not already fixed. This is pretty complex to do and requires quite a bit of setup, especially for users not familiar with the `dotnet/runtime` repo.

Being able to just do this directly into sharplab would make this incredibly easier and more accessible 🚀

Is this something that would be difficult to add, or would it be relatively easy for someone with experience with this codebase?

**Comment by ashmind** [comment]
I think it would be possible to do, but would be a build step instead of a code thing. It makes sense -- I will look into it, but can't promise any specific timelines -- which is probably clear by the delay in this response :)

**Comment by Sergio0694** [comment]
Of course, wasn't expecting any exact timeline! Thank you for looking into this at all though!
And happy holidays, while I'm at it, thank you for your great work on sharplab! 🥳

**Comment by Sergio0694** [comment]
Just thought I'd touch base again to ask whether there's still interest in this, and if some help is needed to make this happen, and if so what could we do to help. Having this feature would greatly simplify opening codegen related issues in the runtime (no longer having to clone and build the runtime to check things with Disasmo, just open sharplab instead), and it'd reduce the number of new issues being open in the runtime by people using the main branch on sharplab accidentally thinking it's the latest from the runtime 😄

I know @egorbo was also looking into adding support for automatically grabbing a nightly build from Disasmo to avoid having to use a local build of the runtime, so maybe there's some work here that could be shared across the two projects, or at least knowledge on how to set this up that could be shared to help? 🙂

**Comment by MichalPetryka** [comment]
@ashmind Has there been any progress on this? You could use daily builds from https://github.com/dotnet/installer for example.
